facet-core/src/types/value.rs

```diff
@@ -723,6 +767,7 @@ pub struct ValueVTableBuilder<T> {
     drop_in_place: Option<DropInPlaceFn>,
     invariants: Option<InvariantsFnTyped<T>>,
     parse: Option<ParseFnTyped<T>>,
+    to_string: Option<ToStringFnTyped<T>>,
```

Comment:
The standard ToString is just an extension trait implemented on anything that implements `Display`, right? I'm not clear on how this is actually different from Display.

If this is actually `serialize_to_string` (and `deserialize_from_string`), as a version of `ToString` / `FromStr` that can be used for text formats like JSON with no "date types" etc., then I think we should consider naming it differently.